test/type_traits.cpp

```diff
@@ -152,13 +152,17 @@ TEST_CASE("is_member_object_pointer", "[meta.unary.cat]") {
   class C {};
   CHECK(std::is_base_of<slb::true_type,
                         slb::is_member_object_pointer<int C::*>>::value);
+
+  { CHECK_FALSE(slb::is_member_object_pointer<int (C::*)() noexcept>::value); }
```

Comment:
How come the added tests are not `is_base_of` checks?

Reply:
Based on the `is_null_pointer` tests below. I _suppose_ when I did those I figured checking the base characteristic once was enough since there are no specializations. Will fix.